ts/features/itwallet/common/components/ItwCredentialCard/ItwCredentialCard.tsx

```diff
@@ -129,9 +140,13 @@ export const ItwCredentialCard = ({
             {getCredentialNameFromType(credentialType, "").toUpperCase()}
           </IOText>
           {statusTagProps && <Tag forceLightMode {...statusTagProps} />}
-          {!statusTagProps && isNewItwRenderable && (
-            <ItwCardValidityCheckMark />
-          )}
+          {!statusTagProps &&
+            isNewItwRenderable &&
+            (isMultiCredential ? (
+              <ItwCardMultiCredentialBadge />
+            ) : (
+              <ItwCardValidityCheckMark />
```

Comment:
`<ItwCardValidityCheckMark />` was recently removed. Probably it's just a matter or merging `master`.